ibis/pandas/execution/tests/test_operations.py

```diff
@@ -839,6 +839,18 @@ def test_intersect(client, df1, intersect_df2):
     tm.assert_frame_equal(result, expected)
 
 
+def test_except(client, df1, intersect_df2):
```

Comment:
let's call this difference as its a much more common name than except

Reply:
Ok sounds good, I'll update the names for everything